GTG/gtk/browser/browser.py

```diff
@@ -117,7 +130,7 @@ def __init__(self, requester, vmanager):
         self.activetree.register_cllbck('node-deleted-inview',
                                         self._update_window_title)
         self._update_window_title()
```

Comment:
Yes, this could find its place in my plan, can you please create a bug explaining the issue in more detail? We will be able to discuss the whole matter in depth there.
Thanks